packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/download-sample.md

```diff
@@ -0,0 +1,63 @@
+### Download the sample React application
+
+To view a simple example of a React application, clone the Auth JS repository and follow the set up procedure:
+
+#### Clone the Auth JS repository
+
+Clone the Auth JS repository and navigate to the top-level folder.
+
+```bash
+git clone https://github.com/okta/okta-auth-js.git
+cd okta-auth-js
+```
+
+#### Install the dependencies
+
+Install the dependencies in the `okta-auth-js` folder:
+
+```bash
+yarn install
+```
+
+#### Add  a configuration file
+
+Create and add a configuration file, `testenv`, to the `okta-auth-js` root folder, which contains the configuration values for your Okta app integration. See [Create an app integration](#create-an-app-integration) for details on these values.
```

Comment:
```suggestion
Create and add a configuration file (`testenv`) to the `okta-auth-js` root folder that contains the configuration values for your Okta app integration. See [Create an app integration](#create-an-app-integration).
```

Reply:
This sentence was not cast well. I broke it up into two: 

"Create and add a configuration file (`testenv`) to the `okta-auth-js` root folder. The `testenv` file contains the configuration values for your Okta app integration."